public/sitemap.xml

```diff
@@ -0,0 +1,64 @@
+<!-- public/sitemap.xml -->
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/feedback</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/providers</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/data-service-providers</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/contact-research-support</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/research-support</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/providers?serviceType=Find</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/providers?serviceType=Recruit</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/providers?serviceType=Follow-Up</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://www.accessibility-services.co.uk/certificates/national-institute-for-health-and-care-research/</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/terms-and-conditions</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/privacy</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/cookie-policy</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk/accessibility</loc>
+    <lastmod></lastmod>
+  </url>
+  <url>
+    <loc>https://www.nihr.ac.uk/</loc>
```

Comment:
Why is this in the sitemap?  It's a different site?

Reply:
Its at the bottom of the homepage, As an image, I thought in a sitemap you would have to include every site is linked to. I can remove it no problem